src/Features/Core/Portable/SolutionCrawler/SolutionCrawlerWorkspaceEventListener.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Composition;
+using Microsoft.CodeAnalysis.Host;
+
+namespace Microsoft.CodeAnalysis.SolutionCrawler
+{
+    [ExportWorkspaceEventListener(WorkspaceKind.Host), Shared]
```

Comment:
Doesn't this run for all workspaces in VS? If I recall we never really gave the Visual Studio workspace it's own, right?

Reply:
solution crawler never run for all workspaces. one explicitly need to enable solution crawler for a workspace if they want to.